src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -29,23 +34,23 @@
                     <div class="col-8">{{ 'artemisApp.tutorExerciseDashboard.numberOfSubmissionsAssessedByYou' | translate }}</div>
                     <div class="col-4">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</div>
                 </div>
-                <div class="row mb-1">
+                <div *ngIf="exercise.type !== ExerciseType_PROGRAMMING" class="row mb-1">
```

Comment:
Can't we just use an `ng-container` here?

Reply:
Would also work, but also works like this.